projects/app/src/app/(sidenav)/_layout.tsx

```
@@ -81,36 +127,62 @@ export default function SideNavLayout() {
     {
       children,
-      isFocused,
+      isFocused = false,
       style, // pull out of `...props` and don't pass to <Pressable>
       ...props
     },
     forwardedRef,
   ) => (
     <Pressable {...props} ref={forwardedRef}>
-      <Text className={buttonClass({ focused: isFocused })}>{children}</Text>
+      {typeof children === `function` ? (
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)